### DISCUSSION TITLE
Remove Popup shroud/overlay or set different color.

### DISCUSSION THREAD

**Discussion by SergTomcat** [CommunityToolkit/Maui#824] [Q&A]
When popup appear, is it possible  to change color of overlay or remove it?

For example make, it red:
![image](https://user-images.githubusercontent.com/24654961/207086397-d396be6a-82ad-4e2b-a86b-a82f25aa7ae7.png)

Goal is to make it transparent.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
